lib/mix/lib/mix/tasks/escript.build.ex

```diff
@@ -45,7 +45,7 @@ defmodule Mix.Tasks.Escript.Build do
       Defaults to app name. Set it to `nil` if no application should
       be started.
 
-    * `:embed_elixir` - if `true` embed elixir and its children apps
+    * `:embed_elixir` - if `true` embed `elixir` and its children apps
```

Comment:
If it is the app name, I would go with `:elixir`, but I don't want to bikeshed on this much. :)